src/components/App.js

```diff
@@ -1,13 +1,51 @@
 import React, { Component } from 'react';
-import { connect } from 'react-redux';
+import { Link } from 'react-router';
 import SignIn from '../containers/SignIn';
 
-export class App extends Component {
+const NavigationLink = (props) =>(
```

Comment:
probably could've extracted this into its own file but I figure it's only going to be used here...